File: demo/springboot/demo/src/main/java/com/demo/domain/dto/http/resonse/OssBatchDeleteResponse.java
package com.demo.domain.dto.http.resonse;

import lombok.Data;

import java.util.List;

@Data
public class OssBatchDeleteResponse {
    /** Each id as decimal string in JSON. */
    private List<String> fileIdList;
    private Integer successCount;
    private Integer failCount;
    private String message;
}

File: demo/app/app/src/main/java/com/vectordemo/domain/dto/http/response/OssBatchDeleteResponse.kt
package com.vectordemo.domain.dto.http.response

/** [fileIdList] elements are decimal strings in JSON. */
data class OssBatchDeleteResponse(
    val fileIdList: List<String>? = null,
    val successCount: Int? = null,
    val failCount: Int? = null,
    val message: String? = null
)
